CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py

```diff
@@ -311,12 +313,15 @@ def __init__(self, *args, **kwargs) -> None:
 
         self.auth_token = None
 
-        self.app_id = f"sio-{time.time()}"
+        self.app_id = self.generate_app_id()
```

Comment:
Can you move the generation of the app_id into `EventStreamReader` instead? Thanks

Reply:
But then we won't be able to access it directly from the connector